### DISCUSSION TITLE
non snap or flatpak installation?

### DISCUSSION THREAD

**Discussion by babag1** [WarmUpTill/SceneSwitcher#464] [General]
i downloaded and extracted the 'advanced-scene-switcher' folder but can find no instructions on where to put the folders/files. 

i'm on ubuntu studio 20.04 and obs studio 27. i installed obs via 'sudo apt install' after adding its repository. i didn't see instructions regarding this type of installation. it seemed to all be snap and flatpak.

thanks for any guidance,
babag

**Comment by WarmUpTill** [comment]
Unfortunately it can vary, but the following setup is most likely work (replace "user" with your username:

```
/home/user/.config/obs-studio/plugins/
└── advanced-scene-switcher
    ├── bin
    │   └── 64bit
    │       └── advanced-scene-switcher.so
    └── data
        ├── locale
        │   ├── de-DE.ini
        │   ├── en-US.ini
        │   ├── ru-RU.ini
        │   └── zh-CN.ini
        └── res
            ├── cascadeClassifiers
            │   ├── haarcascade_eye_tree_eyeglasses.xml
            │   ├── haarcascade_eye.xml
            │   ├── haarcascade_frontalface_alt2.xml
            │   ├── haarcascade_frontalface_alt_tree.xml
            │   ├── haarcascade_frontalface_alt.xml
            │   ├── haarcascade_frontalface_default.xml
            │   ├── haarcascade_fullbody.xml
            │   ├── haarcascade_lefteye_2splits.xml
            │   ├── haarcascade_lowerbody.xml
            │   ├── haarcascade_profileface.xml
            │   ├── haarcascade_righteye_2splits.xml
            │   └── haarcascade_upperbody.xml
            └── time.svg
```

**Comment by babag1** [comment]
thanks for the quick reply but this just seems to be a diagram of the folder structure i downloaded and extracted. i don't see 'user' anywhere in it and still don't follow where i should move these folders or files to. sorry if i'm being dumb here. i'm completely new to obs and have never tried to install a plugin in it. there seems to be little info on how to do so so i appreciate your engagement on this. the plugin looks awesome and i'm anxious to try it.

thanks,
babag

________________________________________

**Comment by babag1** [comment]
somewhere on the intertubes i saw something that pointed towards '/lib/obs-plugins' as a location i should try. i copied the 'advanced-scene-switcher' dolder there but it didn't seem to do anything.

thanks again,
babag

________________________________________

**Comment by WarmUpTill** [comment]
Sorry, the first line somehow didn't show (probably messed something up with the backticks).
I updated the previous comment, but basically you have to create this folder `/home/user/.config/obs-studio/plugins/advanced-scene-switcher` and move the plugin there.

**Comment by babag1** [comment]
ah! that makes perfect sense and seems to have worked perfectly as well! anxious to get started now.

thanks again!
babag


________________________________________
